packages/desktop/src/main/SketchesServer.ts

```diff
@@ -85,6 +85,7 @@ export class SketchesServer extends EventEmitter {
         // text: loaded into sketch as string
         '.glsl': 'text',
         '.isf': 'text',
+        '.ply': 'text',
```

Comment:
sneaking this in, would be good if there is a way to import anything unknown as text

Reply:
Good point. I feel like there should be a way to do this with esbuild but a quick search didn't show much. Did you try with `.*: text` ?